src_updater/updater.cpp

```diff
@@ -0,0 +1,234 @@
+/*
+  VitaShell
+  Copyright (C) 2015-2019, TheFloW, Robsdedude, devnoname120
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
```

Comment:
Will remove this since we got authorization to relicense this file, and it's incompatible with the (unadapted) Creative Common license that we use.